js/packages/openinference-instrumentation-bedrock-agent-runtime/src/collector/agentTraceAggregator.ts

```diff
@@ -337,14 +353,20 @@ export class AgentTraceAggregator {
   }: {
     agentChildId: string | null;
     eventType: string;
-    chunkType: string;
+    chunkType: ChunkType | undefined;
     traceData: StringKeyedObject;
   }): void {
     const eventObj =
       getObjectDataFromUnknown({ data: traceData, key: eventType }) ?? {};
-    const chunkObj =
-      getObjectDataFromUnknown({ data: eventObj, key: chunkType }) ?? {};
-    if (agentChildId && chunkObj.agentCollaboratorInvocationOutput) {
+    const chunkObj = chunkType
+      ? (getObjectDataFromUnknown({ data: eventObj, key: chunkType }) ??
+        undefined)
```

Comment:
don't mind this but do you need to fallback to undefined here?


```suggestion
      ? getObjectDataFromUnknown({ data: eventObj, key: chunkType })
```